src/olympia/reviewers/utils.py

```diff
@@ -702,15 +703,18 @@ def process_super_review(self):
         log.info(u'Super review requested for %s' % (self.addon))
 
     def confirm_auto_approved(self):
-        """Confirm an auto-approval decision.
-
-        We don't need to really store that information, what we care about
-        is logging something for future reviewers to be aware of, and, if the
-        version is listed, incrementing AddonApprovalsCounter, which also
-        resets the last human review date to now, and log it so that it's
-        displayed later in the review page."""
-        status = self.addon.status
-        latest_version = self.version
+        """Confirm an auto-approval decision."""
+
+        channel = self.version.channel if self.version else None
```

Comment:
when is `self.version` not defined?  I'd be a bit concerned about passing version=None to `self.log_action` too (the channel would be None so in the else we set version to None).

Reply:
It should never happen, because the action would not be available... I think. I'll double check and look if we need more (or less) checks against this to avoid doing weird things.